### PR TITLE
[Cleanup] Remove old comments from headers, format

### PR DESCRIPTION
**Pull request by thehobojoe**
### Work done
- Remove the `loaded by default?` comment that has been copy-pasted between widgets and gadgets since 2007
- Remove VCS headers from ancient VCS systems
- Remove copyright fields from widgets and gadgets that have author and license information

### CODE REVIEW

File: luarules/gadgets/api_ranks.lua [hunk 1]
```diff
@@ -1,16 +1,13 @@
...
-		desc = "a shared interface to recieve unit ranks",
-		author = "jK",
-		date = "Dec 19, 2007",
+		name    = "ranks api",
+		desc    = "a shared interface to recieve unit ranks",
+		author  = "jK",
+		date    = "Dec 19, 2007",
 		license = "GNU GPL, v2 or later",
-		layer = -990000,
-		enabled = true  --  loaded by default?
```
Comment (Beherith) [inline luarules/gadgets/api_ranks.lua:13]: __comma?

File: luarules/gadgets/api_ranks.lua [hunk 2]
```diff
@@ -1,16 +1,13 @@
...
 function gadget:GetInfo()
 	return {
-		name = "ranks api",
-		desc = "a shared interface to recieve unit ranks",
-		author = "jK",
-		date = "Dec 19, 2007",
+		name    = "ranks api",
+		desc    = "a shared interface to recieve unit ranks",
+		author  = "jK",
+		date    = "Dec 19, 2007",
```
Comment (salinecitrine) [inline luarules/gadgets/api_ranks.lua:7]: I don't see a mention of these whitespace changes in the PR description. Are they intentional? (Personally I prefer single spaces, unaligned, like it was without this change)

Comment (thehobojoe) [reply luarules/gadgets/api_ranks.lua:7]: Nah, just fixing a minor annoyance as I went through them. I'm happy to undo that if others don't agree.

File: luarules/gadgets/api_ranks.lua (hunk 1 above)
Comment (thehobojoe) [reply luarules/gadgets/api_ranks.lua:13]: oops 😄

**Review by sprunk — CHANGES_REQUESTED** [review]
Author field should match what was removed

File: luaui/Widgets/unit_only_fighters_patrol.lua
Comment (Ruwetuin) [inline luaui/Widgets/unit_only_fighters_patrol.lua:1]: -- based on Factory Kickstart by TheFatController (Owen Martindell)

**Comment by Ruwetuin** [comment]
> Author field should match what was removed

they do match, but they note the real name of the author opposed to the nickname written in the getinfo

File: luarules/gadgets/api_ranks.lua (hunk 2 above)
Comment (WatchTheFort) [reply luarules/gadgets/api_ranks.lua:7]: I prefer the single space each side of the equals sign, makes it easier to search for

**Comment by sprunk** [comment]
Real name is the one that matters though (which is why it was at the top and not hidden away in some nonstandard getinfo call)

**Comment by thehobojoe** [comment]
I lean towards keeping the name in the "author" field, simply because that matches other existing widgets and gadgets, and those same authors do not have copyright headers in other files, perhaps because they were removed previously, I'm unsure.

File: luarules/gadgets/api_ranks.lua (hunk 2 above)
Comment (thehobojoe) [reply luarules/gadgets/api_ranks.lua]: The people have spoken, all whitespace changes reverted 
image

**Comment by sprunk** [comment]
Generally the real name should not be removed because it has more legal force than a nick. Any random motherfucker could go to court and claim he's Sprung.

If some widgets have merely a nick then that is a failure of the original widget maker, but does not void his ownership (he will merely have an extra step to prove identity) and is not a reason to remove the information where provided.

**Comment by thehobojoe** [comment]
Given the uncertainty around the coypright headers I'm going to skip that for this PR, it can be revisited at another time.

**Pull request merged**